.github/workflows/php-code-coverage.yml

```diff
@@ -0,0 +1,76 @@
+name: PHP code coverage
+
+on:
+  pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    name: PHP=8.1, WP=6.6, WC=9.1.2
+    env:
+      PHP_VERSION: '8.1'
+      WP_VERSION: '6.6'
+      WC_VERSION: '9.1.2'
```

Comment:
From the changes in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3924, shouldn't WC_VERSION be at least 9.2?

Reply:
I can update this version once I fix the issues there 👍 . If I decide to close that PR, I will keep this version